BrowserKit/Sources/Logger/DefaultLogger.swift

```diff
@@ -6,35 +6,86 @@ import Foundation
 
 public class DefaultLogger: Logger {
     public static let shared = DefaultLogger()
+
     private var logger: SwiftyBeaverWrapper.Type
+    private var sentry: SentryWrapper
     private var fileManager: LoggerFileManager
 
+    public var crashedLastLaunch: Bool {
+        return sentry.crashedLastLaunch
+    }
+
     init(swiftyBeaverBuilder: SwiftyBeaverBuilder = DefaultSwiftyBeaverBuilder(),
+         sentryWrapper: SentryWrapper = DefaultSentryWrapper(),
          fileManager: LoggerFileManager = DefaultLoggerFileManager()) {
         self.fileManager = fileManager
         self.logger = swiftyBeaverBuilder.setup(with: fileManager.getLogDestination())
+        self.sentry = sentryWrapper
+    }
+
+    public func setup(sendUsageData: Bool) {
+        sentry.setup(sendUsageData: sendUsageData)
     }
 
     public func log(_ message: String,
                     level: LoggerLevel,
                     category: LoggerCategory,
+                    extra: [String: String]? = nil,
+                    description: String? = nil,
                     sendToSentry: Bool = false,
```

Comment:
We can infer if the log should be sent to sentry by the log level. If it's fatal we always send it. If it's lower we could also send a breadcrumb.